src/Library/demos/Web View/main.blp

```diff
@@ -0,0 +1,69 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  maximum-size: 700;
+
+  Box {
+    orientation: vertical;
+
+    Box {
+      orientation: vertical;
+      margin-top: 12;
+      margin-bottom: 12;
+      spacing: 6;
+
+      Label {
+        label: "Web View";
+
+        styles ["title-1"]
+      }
+
+      Label {
+        label: "Load and display webpages and HTML";
+      }
+    }
+
+    Box controls {
+      spacing: 6;
+
+      Button button_back {
+        icon-name: "arrow1-left-symbolic";
+        tooltip-text: _("Back");
+      }
+
+      Button button_forward {
+        icon-name: "arrow1-right-symbolic";
+        tooltip-text: _("Forward");
+      }
+
+      Entry url_bar {
+        input-purpose: url;
+        hexpand: true;
+      }
+
+      Button button_reload {
+        icon-name: "refresh-symbolic";
+        tooltip-text: _("Reload");
+      }
+
+      Button button_stop {
+        icon-name: "stop-sign-symbolic";
+        tooltip-text: _("Stop");
+      }
+    }
+
+    Adw.Bin container {
```

Comment:
`Bin` is made for subclassing. You can use a simple `Box` here

Reply:
also please add `styles ["view"]` to make it white

Reply:
For some reason the ``WebView`` only works with container widgets that can only have a single child, I tried using ``Box`` but it doesn't work. It also works with ``Clamp`` but I didn't want to clamp the view so I chose ``Bin``. Is there any other container widget I can use that takes only a single child?

Reply:
That's an interesting find. I have no idea why.

I can confirm, even with `Box` and `append`

According to the inspector the WebView isn't even in the tree.